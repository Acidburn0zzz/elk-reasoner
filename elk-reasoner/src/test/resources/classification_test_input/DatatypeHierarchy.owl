Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

EquivalentClasses(:AA DataSomeValuesFrom(:prop rdfs:Literal))
EquivalentClasses(:AB DataSomeValuesFrom(:prop xsd:dateTime))
EquivalentClasses(:AC DataSomeValuesFrom(:prop xsd:dateTimeStamp))
EquivalentClasses(:AD DataSomeValuesFrom(:prop xsd:base64Binary))
EquivalentClasses(:AE DataSomeValuesFrom(:prop xsd:hexBinary))
EquivalentClasses(:AF DataSomeValuesFrom(:prop xsd:anyURI))
EquivalentClasses(:AG DataSomeValuesFrom(:prop owl:real))
EquivalentClasses(:AH DataSomeValuesFrom(:prop owl:rational))
EquivalentClasses(:AI DataSomeValuesFrom(:prop xsd:decimal))
EquivalentClasses(:AJ DataSomeValuesFrom(:prop xsd:integer))
EquivalentClasses(:AK DataSomeValuesFrom(:prop xsd:nonNegativeInteger))
EquivalentClasses(:AL DataSomeValuesFrom(:prop rdf:PlainLiteral))
EquivalentClasses(:AM DataSomeValuesFrom(:prop xsd:string))
EquivalentClasses(:AN DataSomeValuesFrom(:prop xsd:normalizedString))
EquivalentClasses(:AO DataSomeValuesFrom(:prop xsd:token))
EquivalentClasses(:AP DataSomeValuesFrom(:prop xsd:Name))
EquivalentClasses(:AQ DataSomeValuesFrom(:prop xsd:NMTOKEN))
EquivalentClasses(:AR DataSomeValuesFrom(:prop xsd:NCName))
EquivalentClasses(:AS DataSomeValuesFrom(:prop rdf:XMLLiteral))
)