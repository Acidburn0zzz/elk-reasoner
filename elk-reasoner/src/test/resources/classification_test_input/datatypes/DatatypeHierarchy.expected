Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

SubClassOf(:Literal		owl:Thing)
SubClassOf(:DateTime		:Literal)
SubClassOf(:DateTimeStamp	:DateTime)
SubClassOf(:Base64Binary	:Literal)
SubClassOf(:HexBinary		:Literal)
SubClassOf(:AnyURI		:Literal)
SubClassOf(:Real		:Literal)
SubClassOf(:Rational		:Real)
SubClassOf(:Decimal		:Rational)
SubClassOf(:Integer		:Decimal)
SubClassOf(:NonNegativeInteger	:Integer)
SubClassOf(:PlainLiteral	:Literal)
SubClassOf(:String		:PlainLiteral)
SubClassOf(:NormalizedString	:String)
SubClassOf(:Token		:NormalizedString)
SubClassOf(:Name		:Token)
SubClassOf(:NMToken		:Token)
SubClassOf(:NCName		:Name)
SubClassOf(:XMLLiteral		:Literal)
)